policy/modules/contrib/policykit.te

```diff
@@ -152,8 +152,6 @@ optional_policy(`
 # polkit_auth local policy
 #
 
-allow policykit_auth_t policykit_var_lib_t:dir watch_dir_perms;
```

Comment:
Actually, this one needs to stay. `policykit_auth_t` is allowed to manage `policykit_var_lib_t` files, but not directories. I double-checked the rest, but this seems to be the only such mistake.